.babelrc.js

```diff
@@ -0,0 +1,20 @@
+const presets = [
+  [
+    '@babel/preset-env',
+    {
+      targets: {
+        node: '8',
```

Comment:
@esakal you bumped the babel version to node 8 (which is probably a good thing, we might want to bump it to v10), but didn't update the README about min version

Reply:
I chose v8 since [Electron v2](https://electronjs.org/releases/stable?version=2&page=4) is using Node 8.9.3. Electron V5 already support Node 12.0.0 so we can consider bumping node target to v10 but we will then lose support for apps running in Electron v2.0.8 (published just a few months ago). I might be mistaken but this is what I learned from Electron release page.

Reply:
@esakal 👍